src/Microsoft.DotNet.Build.CloudTestTasks.net45/Microsoft.DotNet.Build.CloudTestTasks.net45.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <NuGetRuntimeIdentifier>win</NuGetRuntimeIdentifier>
+    <ImportedProjectRelativePath>..\Microsoft.DotNet.Build.CloudTestTasks\</ImportedProjectRelativePath>
+    <ProjectGuid>{17C66BCE-EB35-44CA-893C-8AAFB67708E9}</ProjectGuid>
+  </PropertyGroup>
+  <Import Project="$(ImportedProjectRelativePath)Microsoft.DotNet.Build.CloudTestTasks.csproj" />
+  <ItemGroup>
+    <Reference Include="Microsoft.Build.Tasks.v4.0" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="mscorlib" />
```

Comment:
Oh, nevermind; based on the filename I think this is "read all that code, but compile against net45 instead of corefx".